velox/expression/tests/SimpleFunctionTest.cpp

```diff
@@ -1540,6 +1547,13 @@ TEST_F(SimpleFunctionTest, noThrow) {
   VELOX_ASSERT_THROW(
       (evaluateOnce<int64_t, int64_t>("try(no_throw(c0))", 6)),
       "Input must not be 6");
+
+  // Error reported via Status by callNullable.
+  VELOX_ASSERT_THROW(
+      (evaluateOnce<int64_t, int64_t>("no_throw(c0)", std::nullopt)),
+      "Input cannot be NULL");
+  result = evaluateOnce<int64_t, int64_t>("try(no_throw(c0))", std::nullopt);
+  EXPECT_EQ(std::nullopt, result);
```

Comment:
Would you test the other cases for `callNullable` and `callNullFree` as `call` including `Errors reported by throwing exceptions` etc? Perhaps, we could add separate unit tests for them.

Reply:
+1